doc/networking.md

```diff
@@ -52,15 +52,34 @@ This means that a route from the `azure-iot-edge` subnet to the internet must ex
 
 ### Configuring `azure-iot-edge`
 
-This network name is configured via the `moby_runtime` section of the `iotedged`'s config file (`/etc/iotedge/config.yaml):
+This network is configured via the `moby_runtime` section of the `iotedged`'s config file (`/etc/iotedge/config.yaml):
 
 ```yaml
 moby_runtime:
   uri: "unix:///var/run/docker.sock"
   network: "azure-iot-edge"
 ```
 
-Any changes to the specific settings of this network must be made out of band, via the Moby Engine.
+Additional container network configuration such as enabling IPv6 networking and providing the IPAM settings can be achieved by specifying the relevant configuration in the network settings.
+
+```yaml
+moby_runtime:
+  uri: "unix:///var/run/docker.sock"
+  network:
+    name: "azure-iot-edge"
+    ipv6: true
+    ipam:
```

Comment:
Since this specifically corresponds to the `.ipam.config` field of network config rather than `.ipam` itself, do we want to call it `.ipam_config` instead?

Reply:
Yeah, I was debating this. I'm more inclined to add a config subnode and go with that instead. This is so that in case we wish to allow customer's to specify values for the other properties of the docker Ipam config, we can do that easily.

Reply:
Yeah, that's fine.